bk-config/bin/create_known_hosts_file.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+if [ ! -e /usr/lib/sage/certca/sage_beekeeper_ca.pub ] ; then
+    echo CA public key not found!
```

Comment:
quotes in your `echo`

`echo "CA public key not found!"`

Reply:
thx